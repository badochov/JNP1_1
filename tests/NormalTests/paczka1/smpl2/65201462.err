Error in line 5: S339 A188 21,8
Error in line 15: o7Eanm8QLH S451 49,2
Error in line 23: VdU A188 27,2
Error in line 37: 2dxsIRHln S849 26,8
