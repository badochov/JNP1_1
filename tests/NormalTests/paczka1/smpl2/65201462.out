o7Eanm8QLH S 31,4
2TFo A 143,2 S 12,4
QlUucnGsn A 193,6 S 182,1
S339 S 328,9
VdU S 11,6
o7Eanm8QLH S 31,4
A188 336,8
S451 468,0
S849 98,4
